src/helpers/ramlParser.js

```diff
@@ -93,21 +91,45 @@ function _parseRAML(file, source, dest, baseUri, listTraits, cb) {
     const result = toRAML(data);
 
     creator.createFilesSync(dest, result, 'utf-8');
-
-    cb('Generation went fine');
+    return cb('Generation went fine');
   }, (err) => {
-
     if(err && err.message && err.message.indexOf('ECONNREFUSED') !== -1) {
-      logger.warning(`Couldn't download traits, retrying...`);
+      logger.warning('Couldn\'t download traits, retrying...');
```

Comment:
That message is misleading :) Remove word retrying or say that we will retry 3 times :)